src/validation/fields/helper/date.validation.helper.ts

```diff
@@ -43,6 +44,41 @@ export const conditionalDateValidations = (dateContextWithCondition: dateContext
   ];
 };
 
+export const conditionalHistoricalBODateValidations = (trustDateContext: dateContextWithCondition) => [
+  setValidateCeasedDateFlag(trustDateContext),
+  ...conditionalDateValidations(trustDateContext),
+];
+
+// export const setValidateCeasedDateFlag = (trustDateContext: dateContextWithCondition) => [
```

Comment:
Had two implementations - one as custom validation, one as an anonymous function. Will delete before moving the PR out of draft.